### DISCUSSION TITLE
Entity operators crashed for not able find cluster CA cert secret

### DISCUSSION THREAD

**Discussion by misurin** [strimzi/strimzi-kafka-operator#5250] [Q&A]
Hi,

We're running the entity operators version 0.22.1 and facing the following error with both of the 2 entity operators.
For the user operator, it threw an exception telling the secret containing cluster CA cert is not found. I have checked the secret, it's still there for a few months.

![image](https://user-images.githubusercontent.com/45362683/124371755-184cb580-dcaf-11eb-9b2f-7e0c4da318b8.png)

This kube config file missing error then loads to topic operator also crashes.

Is there any related configuration that we should know about?

We're also facing another issue with the topic operator, not sure if it connects with the issue above, but it takes very long for the topic to be created (~30mins) after KafkaTopic resource creation. We have around 50 topics in the cluster.

**Comment by scholzj** [comment]
The operators do not use Kubeconfig file - they should use injected service account information. So that should be what you should look into I guess, why did you Kube cluster not injected it into the pods.
